live-examples/css-examples/table/border-collapse.html

```diff
@@ -0,0 +1,28 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-collapse">
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_one" class="language-css">border-collapse: collapse;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_two" class="language-css">border-collapse: separate;</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+</section>
+
+<div id="output" class="output large hidden">
+    <section id="default-example" class="default-example">
+      <table id="example-element" class="transition-all">
```

Comment:
nit: 4-space indentation (although I have a feeling we're not very consistent about this in HTML files anyway).